packages/flutter/lib/src/widgets/gesture_detector.dart

```diff
@@ -130,8 +131,10 @@ class GestureRecognizerFactoryWithHandlers<T extends GestureRecognizer> extends
 ///
 /// {@tool sample}
 ///
-/// This example turns the light bulb yellow when the "turn lights on" button is
-/// tapped by setting the `_lights` field:
+/// This example of a [Container] containing a black light bulb wrapped in a
+/// [GestureDetector].
+/// It turns the light bulb yellow when the "turn lights on" button is tapped
+/// by setting the `_lights` field:
```

Comment:
Would be cool to add a reference to the animation above to this line since that animation is exactly showing that.

Reply:
What animation do you refer to? Of turning the light bulb on? Or do you mean the ink splash on line 127?

Reply:
I think @goderbauer is referring to the embedded animation above on l#130. It shows the code in use. :)

Reply:
Yes, I am taking about the animation linked in line 130 above: https://flutter.github.io/assets-for-api-docs/assets/widgets/gesture_detector.mp4